tests/test_top_level_editable.py

```diff
@@ -20,29 +20,14 @@ def mocked_repository():
     return MockedPyPIRepository(["--no-index"])
 
 
-@pytest.mark.parametrize(
-    ("input", "expected"),
-    (
-        (tup)
-        for tup in [
-            (
-                [
-                    os.path.join(
-                        os.path.dirname(__file__),
-                        "test_data",
-                        "packages",
-                        "small_fake_with_deps",
-                    )
-                ],
-                ["small-fake-a"],
-            )
-        ]
-    ),
-)
 def test_editable_top_level_deps_preserved(
-    base_resolver, mocked_repository, from_editable, input, expected
+    base_resolver, mocked_repository, from_editable
 ):
-    input = [from_editable(line) for line in input]
+    package_path = os.path.join(
+        os.path.dirname(__file__), "test_data", "packages", "small_fake_with_deps"
+    )
```

Comment:
```python
package_path = os.path.join(PACKAGES_PATH, "small_fake_with_deps")
```
We can use `tests.constants.PACKAGES_PATH` here.

Reply:
Done.